diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index bb409d1ec5f10a0f05c7bb0e6b12f14a13d96403..0000000000000000000000000000000000000000
diff --git a/dist/build/parser.js b/dist/build/parser.js
index f568d68ba9c91cdaa93bcc24403df18761c8bb45..25d480f0a0de80669785a7da798ee10514a2137a 100644
--- a/dist/build/parser.js
+++ b/dist/build/parser.js
@@ -670,8 +670,5 @@ if (typeof require !== 'undefined' && typeof exports !== 'undefined') {
     exports.Parser = parser.Parser;
     exports.parse = function () { return parser.parse.apply(parser, arguments); };
     exports.main = () => { };
-    if (typeof module !== 'undefined' && require.main === module) {
-        exports.main(process.argv.slice(1));
-    }
 }
 //# sourceMappingURL=parser.js.map
